carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+#  Copyright (C) 2018-2020 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# Docker Compose Spec Version
+version: '2'
+
+services:
+  roscore:
+    image: usdotfhwastol/carma-base:CARMASystem_3.3.0
+    network_mode: host
+    container_name: roscore
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.51
+    volumes:
+      - /opt/carma/.ros:/opt/carma/.ros
+    restart: always
+    command: roscore
+
+  messenger:
+    image: usdotfhwastol/carma-messenger-core:1.0.0
+    network_mode: host
+    container_name: carma-messenger-core
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.51
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/opt/carma/.ros
+    command: wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma-messenger-docker.launch
+
+  cohda_dsrc_driver:
+    image: usdotfhwastol/carma-cohda-dsrc-driver:CARMASystem_3.3.0
```

Comment:
same question here

Reply:
good catch. my bad the messenger layout is just way different than the other repos